### PR TITLE
Update spec harness before team

### PR DESCRIPTION
**Pull request by lkriffell**
Add all class methods to the stat_tracker class to test the spec harness

### CODE REVIEW

File: lib/game_statistics.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@ def all_games
 
   def total_score
     all_games.map do |games|
-      games.home_goals + games.away_goals
+      games.home_goals.to_i + games.away_goals.to_i
```
Comment (Callbritton) [inline lib/game_statistics.rb:21]: I see that we had to set these back to .to_i due to the removal of the numeric converter (right?)

File: lib/game_team_data.rb [hunk 2]
```diff
@@ -34,7 +34,7 @@ def initialize()
   end
 
   def self.create_objects
-    table = CSV.parse(File.read('./data/dummy_file_game_teams.csv'), headers: true, converters: :numeric)
```
Comment (Callbritton) [inline lib/game_team_data.rb:37]: I see we removed all of the converters due to the conflicts with the spec harness that we discussed.

File: test/league_statistics_test.rb [hunk 3]
```diff
@@ -26,74 +26,74 @@ def test_find_best_and_worst_offense
...
     assert_equal expected, @league_statistics.average_goals_by_id
   end
 
   def test_goals_by_away_id
     expected = {
-                3=>5,
-                6=>12,
-                5=>1,
-                17=>1
+                "3"=>5,
```
Comment (Callbritton) [inline test/league_statistics_test.rb:84]: Had to change all of these over to strings!

**Comment by Callbritton** [comment]
Reviewed by Chris Allbritton

File: lib/game_statistics.rb (hunk 1 above)
Comment (lkriffell) [reply lib/game_statistics.rb:21]: yep!

**Pull request merged**